apps/imaginary.sh

```diff
@@ -87,7 +87,7 @@ This will most likely clear a lot of space! Also, pre-generated previews are not
             rm -rfv "$NCDATA"/appdata_*/preview
             print_text_in_color "$ICyan" "Scanning Nextclouds appdata directory after removing all previews. \
 This can take a while..."
-            nextcloud_occ files:scan-app-data preview -vvv
+            nextcloud_occ files:scan-app-data -vvv
```

Comment:
is this not working or why did you remove it again?

Reply:
the trick is to remove only the content and not the folder itself

Reply:
OK, gotcha. :+1: